apps/api/src/app/environments-v1/usecases/output-renderers/render-email-output.usecase.ts

```diff
@@ -19,17 +15,30 @@ export class RenderEmailOutputUsecase {
 
   @InstrumentUsecase()
   async execute(renderCommand: RenderEmailOutputCommand): Promise<EmailRenderOutput> {
-    const { body, subject } = emailControlZodSchema.parse(renderCommand.controlValues);
+    const { body, subject } = renderCommand.controlValues;
 
-    if (isEmpty(body)) {
-      return { subject, body: '' };
+    if (!body || typeof body !== 'string') {
```

Comment:
Nit picky comment: Since tip tap processing takes place if the body exists, it is more readable to change the logic to 

let renderedBody = '';

``` if (body) {
  const expandedMailyContent = this.transformMailyDynamicBlocks(body, renderCommand.fullPayloadForRender);
  const parsedTipTap = await this.parseTipTapNodeByLiquid(expandedMailyContent, renderCommand);
  renderedBody = await this.renderEmail(parsedTipTap);
}

return { subject, renderedBody }
```